lib/kafka/schema_registry/service.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+require_relative 'configuration'
+
+module Kafka
+  module SchemaRegistry
+    class Service < Common::Client::Base
+      configuration SchemaRegistry::Configuration
+
+      def fetch(id)
+        data = get("/schemas/ids/#{id}", idempotent: true)
+        data.fetch('schema')
+      end
+
+      def register(subject, schema)
```

Comment:
We won't be able to ever _create_ a schema in the schema registry. In fact, I believe any sort of POST and PUT request are disabled so we might want to remove these method to avoid confusion. 

Actually, you could reach out to the Event Bus team and ask if some non-destructive post or put methods are allowed such as in the `check`, `compatibility_issues` and `compatible?` methods.